jenkins/github_helpers.py

```diff
@@ -333,6 +332,10 @@ def compare_pr_differnce(self, txt):
             if req['new_version'] and req['old_version']:  # if both values exits then do version comparison
                 old_version = Version(req['old_version'])
                 new_version = Version(req['new_version'])
+
+                # skip, if the package location is changed in txt file only and both versions are same
+                if old_version == new_version:
+                    continue
```

Comment:
Sometimes only the location of the package is changed within the `txt` file, which is being reported currently as a downgrade. So skip if the version is the same. 

Example PR comment: https://github.com/openedx/repo-tools/pull/327#issuecomment-1362383659